data/data-pipeline/data_pipeline/etl/sources/ejscreen_areas_of_concern/etl.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+
+from data_pipeline.etl.base import ExtractTransformLoad
+from data_pipeline.utils import get_module_logger
+
+logger = get_module_logger(__name__)
+
+
+class EJScreenAreasOfConcernETL(ExtractTransformLoad):
+    def __init__(self):
+        self.OUTPUT_PATH = self.DATA_PATH / "dataset" / "ejscreen_areas_of_concern"
+
+        # TO DO: Load from actual source; the issue is that this dataset is not public for now
+        self.LOCAL_CSV_PATH = self.DATA_PATH / "local"
+        self.GEOID_CBG_FIELD_NAME = "GEOID10_CBG"
```

Comment:
Should just use `self.GEOID_FIELD_NAME` I think

Reply:
Makes sense!